composer.json

```diff
@@ -10,6 +10,7 @@
     ],
     "require": {
         "php": "^7",
+        "knplabs/knp-menu-bundle": "^2.2",
```

Comment:
Er is ook een `2.3`, heb je deze ook geprobeerd? Zou theoretisch moeten kunnen en in die release wordt niet gesproken over BC breaks oid https://github.com/KnpLabs/KnpMenuBundle/releases

Reply:
Heb deze versie constraint overgenomen uit de Sonata Admin Bundle. Die zorgde er nu voor dat de menu bundle in projecten terecht kwam. Ik wil deze constraint niet verhogen, want misschien werkt in projecten dan de Sonata Admin opeens niet meer correct :grin: